plugins/modules/monitoring/sensu/sensu_silence.py

```diff
@@ -181,7 +181,7 @@ def clear(module, url, check, subscription):
             )
 
         try:
-            json_out = json.loads(response.read())
+            json_out = json.loads(response.read().decode('utf-8'))
```

Comment:
I would use `to_native` like 

```suggestion
			from ansible.module_utils._text import to_native
			...
            json_out = json.loads(to_native(response.read()))
```

Reply:
@Akasurde thanks for your input. `to_native` is a much better approach. Pushed another commit.